audit-service/README.md

```diff
@@ -18,8 +18,31 @@ limitations under the License.
 
 # Audit Service
 
-The Audit service accepts incoming messages on the `error` and `success` Kafka topics, these messages contain all the details of the initial request and any other relevant information. This information will be passed to any local audit services that have
-been implemented. This service does not have any output Kafka topics.
+The Audit service is an implementation of the [AuditService Interface](src/main/java/uk/gov/gchq/palisade/service/audit/service/AuditService.java) which audits any incoming messages.
```

Comment:
This sentence kind of doesnt make a lot of sense... especially as the first sentence in the readme
Id suggest something like
```suggestion
The Audit Service can have multiple implementations, configured in the [application.yaml](src/main/resources/application.yaml), but simply, all implementations recieve, and log Audit Messages from the other Palisade services
```

Reply:
Updated